script/release/dash_createAndDeploy.sh

```diff
@@ -0,0 +1,52 @@
+# removing dashbuilder artifacts from local maven repo (basically all possible SNAPSHOTs)
```

Comment:
Do we want to have these dash and uf scripts here? Shouldn't they be in the dashbuilder and uberfire repos?